engo.go

```diff
@@ -118,6 +116,16 @@ func Run(o RunOptions, defaultScene Scene) {
 
 	opts = o
 
+	// TODO: We may need to check this later on for platform
+	InitGl()
```

Comment:
Engo is a game engine not a game library. [Difference between the two](http://gamedev.stackexchange.com/questions/10770/whats-the-difference-between-a-library-and-an-engine)